v2.6/getting-started/kubernetes/upgrade/migrate.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Migrating from TPR to CRD
```

Comment:
I think we should just link to the existing upgrade markdown document rather than duplicate the content here.

Reply:
I think it is a much better experience to bring it forward into the doc site, so that's why I did this, but if you want we can discuss it further....

Reply:
Let's chat more.

We made an explicit decision a long time ago not to bring this into the docs site, which is why it is where it is.

There are other docs, github issues, etc linking to the existing document, so we can't just delete it without doing work - I really hate having two sources of truth. Unless there's a compelling reason beyond "it gets the same theme as the other docs" then I think we should leave it as is.

Reply:
I agree about having two sources....but I think the source should be exposed on the doc site instead. 😄  In any case, we can chat a little later about this as there are some open questions still about the paths in any case.

Reply:
> .but I think the source should be on the doc site instead. 

That might be, but for now it is not, and I don't think it is worth changing given all the things that reference it.

Reply:
Well, I did make some I think substantial improvements to the content when I moved it into the doc site. But to your point, I've gone ahead and changed the v2.6 paths that reference it. Once we release v3.0.0, this content will soon fade away into the mists of memory.

Reply:
Ok. We should discuss tomorrow, because I thought we had agreed to leave it where it was.

Reply:
> But to your point, I've gone ahead and changed the v2.6 paths that reference it. 

Those aren't all the paths that reference it.  There are paths in kops, canal, and other external repositories.